src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml.MD5

```diff
@@ -0,0 +1 @@
+3a1eb5f9b9b4b32e568300da4b135171
```

Comment:
Please re-run `ant generate-i18n` to update this file (along with CmdConstants_en.properties and CmdConstants.java) to match the current Commands.cmd.xml.

Reply:
Done! Should all be up to date now